python/tank/template.py

```diff
@@ -396,26 +398,24 @@ class TemplatePath(Template):
     """
     Class for templates for paths.
     """
-    def __init__(self, definition, keys, root_path, name=None):
+    def __init__(self, definition, keys, root_path, name=None, root_paths_all_os=None):
```

Comment:
root_paths_all_os feels clunky when I read it! - how about something like os_root_paths or platform_root_paths (as we use the term platform elsewhere)?